examples/scatter/scatter_colorslice.py

```diff
@@ -0,0 +1,36 @@
+"""
+Scatter Plot
+============
+Example showing color slice for scatter plot.
+"""
+
+# test_example = true
+
+from fastplotlib import Plot
+import numpy as np
+from pathlib import Path
+
+from wgpu.gui.offscreen import WgpuCanvas
+from pygfx import WgpuRenderer
+
+canvas = WgpuCanvas()
+renderer = WgpuRenderer(canvas)
+
+plot = Plot(canvas=canvas, renderer=renderer)
+
+data_path = Path(__file__).parent.parent.joinpath("data", "scatter.npy")
```

Comment:
let's just use the iris dataset for this, it's small so we can just include it in the repo: https://archive.ics.uci.edu/ml/datasets/iris

Reply:
the iris dataset is `dims=(150,4)` which is not supported by `Scatter`, is it okay to just trim a dimension off? 

```
from sklearn import datasets
iris = datasets.load_iris()
iris_data = iris['data'][:,:-1]
```